### PR TITLE
066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HM…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #932

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+for i = 1, #hmi_result_code do
+	local IndicatorValue
+	if (math.fmod(i,2) ~= 0)then
+		IndicatorValue = "PAUSE"
+	else
+		IndicatorValue = "PLAY"
+	end
+
+	Test["TestStep_SetAudioStreamingIndicator_"..hmi_result_code[i].result_code.."_audioStreamingIndicator_" .. IndicatorValue] = function(self)
+	  local corr_id_first_req = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = "IndicatorValue" })
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua]: @istoimenova, Please fix: '"IndicatorValue"' => 'IndicatorValue'

File: test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	  local corr_id_second_req
+
+	  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", { audioStreamingIndicator = "IndicatorValue" })
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua]: @istoimenova, Please fix: '"IndicatorValue"' => 'IndicatorValue'

File: test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+	  :Do(function(_,data) 
+	  	corr_id_second_req = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = "IndicatorValue" })
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua]: @istoimenova, Please fix: '"IndicatorValue"' => 'IndicatorValue'

File: test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+	  	
+	  	local function send_HMI_response()
+	  	  self.hmiConnection:SendResponse(data.id, data.method, hmi_result_code[i].result_code) 
+	  	end
+	  	
+	  	RUN_AFTER(send_HMI_response, 1500)
```
Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua]: @istoimenova As I understand correct here you are responding for first (which should be successfuly) request, but after 1000ms you will get GENERIC_ERROR for first request

File: test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua]: @dboltovskyi, direct call of variable IndicatorValue. Done in 9e38eff8c367b012d67ec11de01249cbf0b690fc

File: test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua]: @dboltovskyi, direct call of variable IndicatorValue. Done in 9e38eff8c367b012d67ec11de01249cbf0b690fc

File: test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua (hunk 3 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua]: @dboltovskyi, direct call of variable IndicatorValue. Done in 9e38eff8c367b012d67ec11de01249cbf0b690fc

File: test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua (hunk 4 above)
Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua]: @OVikhrov, GENERIC_ERROR is returned after default timeout is elapsed(in ini - 10000 msec).

**Pull request merged**